server/pulp/server/content/sources/model.py

```diff
@@ -139,7 +139,10 @@ def load_all(conf_d=None):
             if not os.path.isfile(path):
                 continue
             cfg = ConfigParser()
-            cfg.read(path)
+            list_of_read_files = cfg.read(path)
+            if not list_of_read_files:
+                log.warn("Could not read content source config file at path %s", path)
```

Comment:
On Fri, Oct 23, 2015 at 07:29:54AM -0700, Jeff Ortel wrote:
> Suggest *error* instead of warning because by default I don't think *warning* shows up in syslog.

I believe that warning does show up in syslog, since it is a higher
level than info and info does show up. debug is the only one that should
be missing from syslog by default.